src/lib/watt-time/README.md

```diff
@@ -47,81 +34,136 @@ WATT_TIME_PASSWORD: <your-password>
 WATT_TIME_TOKEN: <your-token>
 ```
 
-### Plugin global config
+2. **Dependencies**: Confirm the installation of all required dependencies, including `luxon` and `zod`. These dependencies are imperative for date-time manipulation and input validation, respectively.
 
-- `base-url`: The URL for the WattTime API endpoint.
+## Usage
 
-### Inputs
+To employ the `WattTimeGridEmissions` plugin, adhere to these steps:
 
-**Required Parameters:**
+1. **Initialize Plugin**: Import the `WattTimeGridEmissions` function and initialize it with optional global configuration parameters.
+
+2. **Execute Plugin**: Invoke the `execute` method of the initialized plugin instance with an array of input parameters. Each input parameter should include a `timestamp`, `duration`, and either `geolocation`, `cloud/region-wt-id`, or `cloud/region-geolocation` information.
```

Comment:
@manushak @jmcook1186 Can you please provide more samples (manifests) for these use cases? It may be useful both for testing and also as documentation for future